.github/workflows/ci.yml

```diff
@@ -55,20 +55,17 @@ jobs:
         ghc-version: ${{ matrix.ghc }}
         cabal-version: ${{ matrix.cabal }}
 
-    - uses: actions/checkout@v3
-
+    # libsodium, and secp256k1 install utilities
     - name: Install build environment
       run: |
         sudo apt-get update
         sudo apt-get -y install libsodium-dev
+        sudo apt-get -y install autoconf automake libtool
 
     - name: Configure Cabal to use libsodium
       run: |
         cp .github/workflows/cabal.project.local .
 
-    - name: "LINUX: Install build environment (for secp256k1)"
-      run: sudo apt-get -y install autoconf automake libtool
-
```

Comment:
INFO: I merged this with the `Install build environment` step

Reply:
Should not be necessary after #288